proto/wechaty/puppet/message.proto

```diff
@@ -25,17 +25,21 @@ enum MessageType {
   MESSAGE_TYPE_ATTACHMENT   = 1;
   MESSAGE_TYPE_AUDIO        = 2;
   MESSAGE_TYPE_CONTACT      = 3;
-  MESSAGE_TYPE_EMOTICON     = 4;
-  MESSAGE_TYPE_IMAGE        = 5;
-  MESSAGE_TYPE_TEXT         = 6;
-  MESSAGE_TYPE_VIDEO        = 7;
-  MESSAGE_TYPE_CHAT_HISTORY = 8;
-  MESSAGE_TYPE_LOCATION     = 9;
-  MESSAGE_TYPE_MINI_PROGRAM = 10;
+  MESSAGE_TYPE_CHAT_HISTORY = 4;
```

Comment:
OOPS! NEVER CHANGE THE ENUM NUMBER IN A PROTO FILE!

THIS WILL BREAK ALL PREVIOUS VERSION COMPATIBILITIES!

Reply:
Done. I was trying to make the message type enum in grpc match with the one in wechaty-puppet. I did not notice the diffrence in order.